homeassistant/components/atag/climate.py

```diff
@@ -0,0 +1,108 @@
+"""Initialization of ATAG One climate platform."""
+from typing import List, Optional
+
+from homeassistant.components.climate import ClimateDevice
+from homeassistant.components.climate.const import (
+    CURRENT_HVAC_HEAT,
+    CURRENT_HVAC_IDLE,
+    DEFAULT_MAX_TEMP,
+    DEFAULT_MIN_TEMP,
+    HVAC_MODE_AUTO,
+    HVAC_MODE_HEAT,
+    HVAC_MODE_OFF,
+    SUPPORT_TARGET_TEMPERATURE,
+)
+from homeassistant.const import ATTR_TEMPERATURE, TEMP_CELSIUS
+from homeassistant.helpers.restore_state import RestoreEntity
+
+from . import DOMAIN, ENTITY_TYPES, AtagEntity
+
+PLATFORM = "climate"
```

Comment:
Please import this constant from climate/init.py.

Reply:
importing once in init.py, does that work now?